app.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <div>
-    <NuxtWelcome />
-  </div>
+    <NuxtPage >Reading Hurdles Application</NuxtPage>
+    <div>
+        <PageHeader />
+    </div>
```

Comment:
Dont put content inside of the NuxtPage - that gets handled automatically by Nuxt router. 

Also, the header should be above the page